docs/.vuepress/theme/Layout.vue

```diff
@@ -16,7 +14,7 @@
           </p>
           <Content />
         </main>
-        <Sidebar :items="sidebarItems" @close-sidebar="closeSidebar" />
+        <Sidebar :items="sidebarItems" />
```

Comment:
Is the `Sidebar` component even used / needed now? It appears not (removing it keeps all functionality, from what I can tell). If so, let's delete it.

Reply:
I believe that would also mean we could delete `SidebarGroup`, `SidebarLink` & `SidebarButton` components.